improver_tests/wxcode/wxcode/test_WeatherSymbols.py

```diff
@@ -1164,18 +1164,20 @@ def test_unmatched_validity_times(self):
         cubes[-1].coord("time").points = cubes[-1].coord("time").points + 3600
 
         msg = (
-            "Weather symbol input cubes are valid at different times"
-            "['probability_of_lwe_snowfall_rate_above_threshold: 1507636800', "
+            "Weather symbol input cubes are valid at different times; \n"
+            "\['probability_of_lwe_snowfall_rate_above_threshold: 1507636800', "
             "'probability_of_lwe_sleetfall_rate_above_threshold: 1507636800', "
             "'probability_of_rainfall_rate_above_threshold: 1507636800', "
             "'probability_of_lwe_precipitation_rate_in_vicinity_above_threshold: 1507636800', "
             "'probability_of_low_and_medium_type_cloud_area_fraction_above_threshold: 1507636800', "
             "'probability_of_low_type_cloud_area_fraction_above_threshold: 1507636800', "
             "'probability_of_visibility_in_air_below_threshold: 1507636800', "
             "'probability_of_lwe_precipitation_rate_above_threshold: 1507636800', "
-            "'probability_of_shower_condition_above_threshold: 1507640400']"
+            "'probability_of_shower_condition_above_threshold: 1507636800', "
+            "'probability_of_lwe_graupel_and_hail_fall_rate_in_vicinity_above_threshold: 1507636800', "
+            "'probability_of_lwe_precipitation_rate_max_above_threshold: 1507640400'\]"
         )
-        with self.assertRaises(ValueError, msg=msg):
+        with self.assertRaisesRegex(ValueError, msg):
```

Comment:
Previously implemented incorrectly. Fixed in this PR, but unrelated to the change being implemented.